Assets/MainMenu.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public void PlayGame() {
+        int currentLevel = SceneManager.GetActiveScene().buildIndex + 1;
+        SceneManager.LoadScene(currentLevel);
```

Comment:
This works based on the adding the scenarios to the BuildSettings in a given order
https://github.com/CleverGameDev/Cleverfall/pull/3/files#diff-8f3c83eb79d0b4aa8cf581a168df971cR7-R13